### ISSUE TITLE
Not available in CRAN any longer?

### ISSUE THREAD

**Issue by itchyshin** [matthiasspeidel/hmi#3]
We have started using this package for our project, but it does not seem to be available via CRAN any longer. Do you have a plan to update?

**Comment by matthiasspeidel** [comment]
Yes, I want to have it back on CRAN, but unfortunately cannot tell when this will happen.
